pkg/queue/activemq/activemq.go

```diff
@@ -0,0 +1,76 @@
+package activemq
+
+import (
+	"context"
+	"log/slog"
+	"net"
+	"strconv"
+	"time"
+
+	"github.com/bugrakocabay/konsume/pkg/config"
+	"github.com/go-stomp/stomp/v3"
+)
+
+type Consumer struct {
+	conn   *stomp.Conn
+	sub    *stomp.Subscription
+	config *config.StompConfig
+}
+
+func NewConsumer(cfg *config.StompConfig) *Consumer {
+	return &Consumer{
+		config: cfg,
+	}
+}
+
+func (c *Consumer) Connect(ctx context.Context) error {
+	slog.Debug("Attempting to connect to ActiveMQ", "host", c.config.Host, "port", c.config.Port)
+	var err error
+	var options = []func(*stomp.Conn) error{
+		//set rw timeout to 2 hours
+		stomp.ConnOpt.HeartBeat(2*time.Hour, 2*time.Hour),
+		stomp.ConnOpt.HeartBeatError(5 * time.Minute),
+		stomp.ConnOpt.Login(c.config.Username, c.config.Password),
+	}
+	var connectionString = net.JoinHostPort(c.config.Host, strconv.Itoa(c.config.Port))
+
+	c.conn, err = stomp.Dial("tcp", connectionString, options...)
+	if err != nil {
+		return err
+	}
```

Comment:
Can you add a log message here, something like;
`slog.Info("Connected to ActiveMQ", "host", c.config.Host, "port", c.config.Port)`

Reply:
I resolved it